pykla/accounts/views.py

```diff
@@ -0,0 +1,65 @@
+from django.shortcuts import render, redirect
+from.forms import RegForm, EditProfile
+from django.contrib.auth.forms import PasswordChangeForm
+from django.contrib.auth import update_session_auth_hash
+from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required
+
+
+# index view
+def index(request):
+    return render(request, 'base.html')
+
+#registration view
+def reg(request):
```

Comment:
@Dru94 use proper function names, `reg` should be `register`, I know there's a comment that says registration view but code should be self documenting and require very little comments